arch/configure.defaults

```diff
@@ -2123,6 +2123,46 @@ NETCDFPAR_BUILD =       CONFIGURE_NETCDFPAR_BUILD
 
 #insert new stanza here
 
+###########################################################
+#ARCH    Linux aarch64, GCC compiler OpenMPI # serial smpar dmpar dm+sm
+#
+DESCRIPTION     =      armclang ($SFC/$SCC): Aarch64
+DMPARALLEL      =
+OMPCPP          =      -fopenmp
+OMP             =      -fopenmp
+OMPCC           =      -fopenmp
+SFC             =      armflang
+SCC             =      armclang
+CCOMP           =      armclang
+DM_FC           =      mpif90
+DM_CC           =      mpicc -DMPI2_SUPPORT
+FC              =      CONFIGURE_FC
+CC              =      CONFIGURE_CC
+LD              =      $(FC)
+RWORDSIZE       =      CONFIGURE_RWORDSIZE
+PROMOTION       =
+ARCH_LOCAL      =
```

Comment:
@mgduda Would we need `-DAARCH64_X86_CORRECTNESS_FIX` in the `ARCH_LOCAL` defines?

Reply:
The flag "-DAARCH64_X86_CORRECTNESS_FIX" was added for GCC to provide a way to achieve same results using GCC on x86 and aarch64 platforms. We don't need this flag here.